### ISSUE TITLE
Upgrade New York City (NYC) neighbourhoods shapes

### ISSUE THREAD

**Issue by nvkelso** [whosonfirst-data/whosonfirst-data#384]
### tl;dr

Status:
- [x] Propose **sources**
- [x] Establish **concordances** between WOF and source
- [x] **Upgrade / add** WOF shapes
- [x] **Add new or missing** features to WOF
- [x] **Deprecate / supersede** WOF features
- [x] **Draft review**
- [x] **Draft revisions** based on review comments
- [x] **Final review**
- [x] **Final revisions** based on review comments
- [x] **Import** into Who's On First
### Details

Sourcing:
- Permissive license?
- Add new src to Who's On First

Upgrade / add:
- `macrohood`
- `neighbourhood`
- `microhood`

Add new or missing features to WOF
- `macrohood`
- `neighbourhood`
- `microhood`

Deprecate / supersede:
- `macrohood`
- `neighbourhood`
- `microhood`

**Comment by nvkelso** [comment]
Possible sources:

- Pediacities: 
  - Most useful source for outer Boroughs. 
  - http://nyc.pediacities.com/Resource/Neighborhood/Concourse 
  - GeoJSON feed: http://catalog.opendata.city/dataset/pediacities-nyc-neighborhoods/resource/91778048-3c58-449c-a3f9-365ed203e914 
- NYC Gov't:
  - Access at http://www1.nyc.gov/site/planning/data-maps/open-data.page:
  - http://www1.nyc.gov/site/planning/data-maps/open-data/dwn-nynta.page (less useful) and 
  - http://www1.nyc.gov/assets/planning/download/zip/data-maps/open-data/nyc_nhoodsnames201408.zip (more useful) includes “neighborhood names” and “neighborhood tabulation areas”.
- Wikipedia: 
  - https://en.wikipedia.org/wiki/List_of_Manhattan_neighborhoods
  -https://en.wikipedia.org/wiki/List_of_Manhattan_neighborhoods#/media/File:Manhattan_neighborhoods.png
  - https://en.wikipedia.org/wiki/List_of_Queens_neighborhoods#/media/File:Queens_neighborhoods_map.png
  - https://en.wikipedia.org/wiki/Rockaway,_Queens#Communities 
  - https://en.wikipedia.org/wiki/Howard_Beach,_Queens#Geography 
  - https://en.wikipedia.org/wiki/Flushing,_Queens#Sections_of_Flushing 
  - https://en.wikipedia.org/wiki/List_of_Brooklyn_neighborhoods 
  - https://en.wikipedia.org/wiki/List_of_Bronx_neighborhoods 
- CUNY:
  - _For all 5 boroughs. Shapes are un-useful._
  - https://www.baruch.cuny.edu/nycdata/population-geography/maps-boroughs.htm

**Comment by nvkelso** [comment]
Looks like **Pediacities** has the better data to start from, most the other sources aren't GeoJSON or SHP (good visual reference), and the official NYC data is bunky.

**Comment by nvkelso** [comment]
**Establish concordances between WOF and source**

- 310 neighbourhoods in Pediacities (and 450 in WOF). 
- 240 match with WOF

**Comment by thisisaaronland** [comment]
![screen shot 2016-08-05 at 11 15 39](https://cloud.githubusercontent.com/assets/12658759/17446392/f9b34ea0-5afd-11e6-91e6-c33e5a230382.png)

"Two Bridges" ... Just in case anyone thought neighbourhoods could be made rational.

**Comment by nvkelso** [comment]
Proposal: move most of the not-matching WOF `neighbourhoods` to `microhood` placetype and then renominate for `neighbourhood` (in draft phase).

**Comment by thisisaaronland** [comment]
Except Gowanus Heights, obviously.

**Comment by nvkelso** [comment]
See also: https://github.com/whosonfirst-data/whosonfirst-data/issues/441

**Comment by nvkelso** [comment]
Files were delivered today and include edits made after reviews with NYC office.

**Comment by nvkelso** [comment]
@stepps00 to do final sanity check on those files, upload to S3, and pass on to Aaron.

**Comment by ZBSingleton** [comment]
Uploading most recent draft of shapes data. File structure remains consistent with previous markets. Data is broken up as follows:

- boroughs - Shape and attribute data for the boroughs of NYC
- deprecated - Shape and attribute data for WOF neighborhoods (polygon data) that were discarded due to a significant event in the data structure or inaccurate starting data
- deprecated_points - Shape and attribute data for WOF neighborhoods (point data) that were discarded due to a significant event in the data structure or inaccurate starting data
- macrohoods - Shape and attribute data for macrohoods that were created from assemblages of connected neighborhoods
- microhoods - Shape and attribute data for small sub-sections of neighborhoods
- no_wof - Neighborhood data (polygons) that did not align with existing WOF records
- wof - Neighborhood data (polygons) that corresponded to existing WOF records
- wof_points - Neighborhood data (points) that corresponded to existing WOF records but were created as point geometries due to hierarchy conflicts with nearby neighborhoods/microhoods

[NYC_20160819_3.zip](https://github.com/whosonfirst-data/whosonfirst-data/files/430821/NYC_20160819_3.zip)

**Comment by stepps00** [comment]
LGTM, I'll do a final sanity check and update S3 shortly.

@ZBSingleton - Noting this as a reminder: we need to discuss the single-part vs. multi-part polygon validation error(s) that caused rendering problems in QGIS.

**Comment by stepps00** [comment]
Over to @thisisaaronland for import.

The geojson files are in the S3 misc bucket (`misc/Issue 384 folder`). No changes to the comments/directions above.

**Comment by stepps00** [comment]
Reassigning @ZBSingleton to update files for NYC.

**Comment by stepps00** [comment]
New York is up-to-date and ready for import. GeoJSON files are in the S3 `misc/Issue 384` folder.

Back to @thisisaaronland for import.

**Comment by stepps00** [comment]
Neighbourhood (and associated) features have been updated in NYC, closing issue.
